thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -48,6 +58,11 @@ struct TDataNodeRegisterResp {
   4: optional TGlobalConfig globalConfig
 }
 
+struct TDataNodeRemoveResp {
+  1: required common.TSStatus status
+  2: optional map<common.TDataNodeLocation, common.TSStatus> nodeToStatus
+  3: optional TGlobalConfig globalConfig
```

Comment:
Why should we need to return TGlobalConfig?

Reply:
it is useless, I has deleted